src/platform_impl/linux/x11/window.rs

```diff
@@ -1306,6 +1289,23 @@ impl UnownedWindow {
         self.set_ime_position_physical(x, y);
     }
 
+    #[inline]
+    pub fn request_user_attention(&self, request_type: Option<RequestUserAttentionType>) {
```

Comment:
I'd personally say that `RequestUserAttentionType` is incorrectly named. It should be `UserAttentionType`, or `UserAttentionRequestType`, the former probably being a bit better.

macOS already uses this, so one could argue it's easier to just keep things the same. But since this introduces breakage in this area anyways, it might be worth evaluating a rename.

Reply:
Think `UserAttentionType` is best as well :) Just copied it over from macOS when I started on this for consistency.